op-node/rollup/derive/deposit_source.go

```diff
@@ -63,3 +64,20 @@ func (dep *UpgradeDepositSource) SourceHash() common.Hash {
 	copy(domainInput[32:], intentHash[:])
 	return crypto.Keccak256Hash(domainInput[:])
 }
+
+// Used for DepositsComplete/ResetDeposits post-deposits transactions.
```

Comment:
Will defer to @protolambda but I lean towards not needing to introduce another source domain, we could reuse `L1InfoDepositSourceDomain`. if we want to differentiate, the input is 64 bytes, we could introduce a version on the right hand side

Right now its `bytes32(0) || block.hash`, we could make it `bytes32(1) || block.hash`

Reply:
proto approved :)